js/src/BrushSelector.ts

```diff
@@ -317,8 +317,10 @@ export class BrushSelector extends BrushMixinXYSelector {
 
         this.brush.extent([[0, 0], [this.width, this.height]]);
         this.syncModelToBrush();
+        this.brushsel = this.d3el.call(this.brush);
```

Comment:
This seems to be executed twice, once in `syncModelToBrush` and once here, should we remove one of those lines @maartenbreddels ?

Reply:
Indeed, this seems not needed.